server/src/main/scala/com/faleknatalia/cinemaBookingSystem/movie/MovieService.scala

```diff
@@ -1,34 +1,36 @@
 package com.faleknatalia.cinemaBookingSystem.movie
 
+import java.net.URI
+
 import slick.jdbc
-import slick.jdbc.JdbcBackend
 import slick.jdbc.PostgresProfile.api._
 import slick.lifted.TableQuery
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class MovieService() {
+class MovieService(db: jdbc.JdbcBackend.Database)(implicit ec: ExecutionContext) {
 
-   def addNewMovie(db: jdbc.JdbcBackend.Database, movieForm: AddMovie, movies: TableQuery[MovieTable])(implicit ec: ExecutionContext) : Future[Unit] = {
-     val movie = Movie(
-       title = movieForm.title,
-       description = movieForm.description,
-       durationInSeconds = movieForm.durationInSeconds,
-       imageUrl = movieForm.imageUrl)
-     db.run(movies += movie).map( _ => ())
-  }
+  private lazy val movies = TableQuery[MovieTable]
```

Comment:
`lazy` ma sens przy tworzeniu kosztownych obiektow, albo wtedy jak chcemy opoznic inicjalizacje. Tutaj raczej bym tego nie stosowal

Reply:
Z ksiazki o Slicku:
"As a rule, use lazy val for TableQuery s and def foreign keys (for consistency
with column definitons)."